spec/layer-spec.yml

```diff
@@ -0,0 +1,1348 @@
+            filters:
+              type: object
+              description: Scene filters to use when adding scenes to attached ProjectLayers. Cannot change.
+            layerRangeStart:
+              type: string
+              format: datetime
+              description: Date to start creating layers for.
+            layerRangeEnd:
+              type: string
+              format: datetime
+              description: Date to stop creating layers for. Must be a multiple of periodDays after activeStart
+            isActive:
+              type: boolean
+              description: |
+                Pausing a smart layer will cause new scenes to no longer be added to associated project layers.
+                Resuming a smart layer will backfill associated project layers, but there may be a delay
+                for processing
+            periodDays:
+              type: long
+              description: |
+                Time period for each created layer to cover. Cannot change.
+            createEmptyLayers:
+              type: boolean
+              description: |
+                Create all associated project layers up front instead of when there is a scene to add to them.
+                Cannot change.
+
+  ProjectSmartLayerPaginated:
+    type: object
+    allOf:
+      - $ref: '#/definitions/PaginatedResponse'
+      - type: object
+        properties:
+          type:
+            type: string
+          features:
+            type: array
+            items:
+              $ref: '#/definitions/ProjectSmartLayer'
+
+  ProjectLayer:
+    type: object
+    properties:
+      type:
+        type: string
+      geometry:
+        $ref: '#/definitions/Geometry'
+      properties:
+        allOf:
+        - $ref: '#/definitions/BaseModel'
+        - $ref: '#/definitions/TimeModelMixin'
+        - type: object
+          properties:
+            name:
+              type: string
+              description: Display name for a layer
+            projectId:
+              type: string
+              format: uuid
+            colorGroup:
+              type: string
+              description: Color hex code for visually identifying layers are related and filtering
+            smartLayer:
+              type: string
+              format: uuid
+            rangeStart:
+              type: string
+              format: datetime
+              description: Start date for scene matching when associated with a smart layer
+            rangeEnd:
+              type: string
+              format: datetime
+              description: End date for scene matching when associated with a smart layer
+      
+  ProjectLayerPaginated:
+    type: object
+    allOf:
+      - $ref: '#/definitions/PaginatedResponse'
+      - type: object
+        properties:
+          type:
+            type: string
+          features:
+            type: array
+            items:
+              $ref: '#/definitions/ProjectLayer'
+
+  AnalysisCreate:
```

Comment:
After conversation with @notthatbreezy , we're just going to add project, layer, and template foreign keys to tool-runs. Will update spec accordingly